### DISCUSSION TITLE
Change the description of a cytoscape subdir

### DISCUSSION THREAD

**Discussion by marimeireles** [cytoscape/cytoscape.js#2988] [Q&A]
Hey, 

I'm the creator of the ipycytoscape(https://github.com/cytoscape/ipycytoscape/) extension and I need to change the description of the repository from 

```
A Cytoscape Jupyter widget
[ipycytoscape.readthedocs.io/en/latest/](https://ipycytoscape.readthedocs.io/en/latest/)
```

to

```
A Cytoscape Jupyter widget
[ipycytoscape.readthedocs.io/en/master/](https://ipycytoscape.readthedocs.io/en/master/)
```

Could someone do that?
Or better yet, give me the permissions to do so?

Sorry to open this here, but seemed like the best place to get in touch with you. If I'm missing some other communication channel please let me know through here or marian(dot)meireles(at)gmail.com

Thank you!

**Comment by maxkfranz** [comment]
Looks like this was resolved

**Reply by marimeireles** [reply]
Hi @maxkfranz, not really
I don't have permissions to change anything that's `settings` related on the ipycytoscape repo. Could you please grant me them?
Thank you.

**Reply by maxkfranz** [reply]
How about now?

**Reply by marimeireles** [reply]
Hey @maxkfranz that worked!
Thank you very much.
